test/binCases/configFile/plugins-precedence/__snapshots__/plugins-precedence.test.js.snap

```diff
@@ -0,0 +1,13 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`plugins-precedence 1`] = `
+"  Asset       Size  Chunks             Chunk Names
+null.js  966 bytes       0  [emitted]  null
+Entrypoint null = null.js
+[0] ./index.js 265 bytes {0} [built]
+[1] ./ok.js 23 bytes {0} [built]
+
+WARNING in configuration
```

Comment:
In Jest we have a helper called `extractSummary` which might be nice here. Separates `summary` from `rest`